sources/main.cpp

```diff
@@ -207,6 +207,27 @@ int main(int argc, char* argv[]) {
     ButtonHandler* buttonHandler = new ButtonHandler(hwFactory->getInterruptHandler(), yioapi);
     qmlRegisterSingletonType<ButtonHandler>("ButtonHandler", 1, 0, "ButtonHandler", &ButtonHandler::getQMLInstance);
 
+    // reset combination was pressed on startup
+    if (buttonHandler->resetButtonsPressed()) {
+        // create marker file
+        QFile file("/firstrun");
```

Comment:
Does this marker file override the wifi configuration at next startup?
Should the reset remove the WiFi configuration?

Reply:
I've added the wifi configuration reset.